deployments/container/nbody/Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM nvcr.io/nvidia/cuda:12.9.1-base-ubuntu22.04 AS builder
+FROM nvcr.io/nvidia/cuda:12.9.1-devel-ubuntu22.04 AS builder
```

Comment:
The reason that I switched to the "base" container here is that the devel image is quite large and causes issues when building this image specifically in CI. This is the reason that we install `cuda-nvcc-12-6` below.

We should probably keep this base and install `cuda-nvcc-12-9` there instead.

Reply:
Thanks for the context. I'm good with using the "base" container if it means we have more stable builds.